.github/workflows/tests.yml

```diff
@@ -133,4 +134,4 @@ jobs:
         run: mage unit
 
       - name: Run integration tests
-        run: mage integration
+        run: python3 -m pytest --reruns 5 -m "not slow" test/integration
```

Comment:
I'm willing to believe that the MacOS ranners on the githab are very slow and I have 2 questions:
- how about testing on self-hosted runners
- at what points do the tests crash (I assume that at some timeouts)

Reply:
Tests crash after 5 reruns of failed tests.

Reply:
Deleted